languages/python/reference/exercise-concepts/variable-length-quantity.md

```diff
@@ -0,0 +1,17 @@
+# Concepts of variable-length-quantity
+
+[Example implementation](https://github.com/exercism/python/blob/master/exercises/variable-length-quantity/example.py)
+
+
+## General
+- Loops: concept required to solve the exercise, `while`, `for`
+- Number comparison: concept required to solve the exercise
+- Number sum: concept required to solve the exercise
+- Lists: knowledge of lists and iteration on lists is required for this exercise
+- `Len()` meethod knowledge is useful
```

Comment:
That should be `len` not `Len`, and it's a function, not a method (or a meethod). Might also be good to flesh out _why_ it's useful in this case.